src/pages/EnablePage.js

```diff
@@ -0,0 +1,37 @@
+// eslint-disable-next-line
```

Comment:
Although it's fairly clear, it would still be nice to have a comment at the start all pages to specify what I going on for the page